libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss

```diff
@@ -0,0 +1,128 @@
+.consultancy-hero-section {
+  background-color: var(--elewa-website-homepage-background-color);
+  width: 100vw;
+  height: 100vh;
+  position: relative;
+  overflow: hidden;
+
+  .consultancy-hero-section-content {
+    display: grid;
+    gap: 20px;
+    margin-left: 260px;
+    margin-top: 200px;
+    position: relative;
+
+    .title {
+      width: 915px;
+      height: 106px;
+      font-family: DMSans-Medium;
```

Comment:
For reuse of fonts, use CSS variables specified in elements/theming/_variables.scss for specifying font family

Reply:
I have made the changes